zip-0317.rst

```diff
@@ -423,6 +425,81 @@ relaying. These policy changes SHOULD NOT be deployed before the changes
+internal wallet in the following PRs:
+
+* https://github.com/zcash/zcash/pull/6527 (fee computation)
+* https://github.com/zcash/zcash/pull/6524 (main implementation)
+* https://github.com/zcash/zcash/pull/6559 (follow-up to #6524)
+* https://github.com/zcash/zcash/pull/6568 (for ``z_shieldcoinbase``)
+* https://github.com/zcash/zcash/pull/6576 (follow-up to #6568)
+* https://github.com/zcash/zcash/pull/6569 (for ``z_mergetoaddress``)
+
+`zcashd` v5.5.0 implemented the `Recommended algorithm for block template construction`_
+in:
+
+* https://github.com/zcash/zcash/pull/6460 (preparation)
+* https://github.com/zcash/zcash/pull/6607 (follow-up to #6460)
+* https://github.com/zcash/zcash/pull/6527 (fee computation)
+* https://github.com/zcash/zcash/pull/6564 (block template construction)
+
+The value used for :math:`block\_unpaid\_action\_limit` by `zcashd`
+can be overridden using the ``-blockunpaidactionlimit`` configuration
+parameter.
+
+`zcashd` v5.5.0 also implemented the change to `Mempool size limiting`_
+to use the ZIP 317 fee for the low fee penalty threshold, in:
+
+* https://github.com/zcash/zcash/pull/6564
+
+As described in section `Transaction relaying`_, nodes MAY drop
+transactions with more unpaid actions than a given limit. From
+`zcashd` v5.6.0, this is controlled by the ``-txunpaidactionlimit``
+configuration option, which defaults to 50 unpaid actions (the
+same default as ``-blockunpaidactionlimit``). This behaviour is
+implemented in:
+
+* https://github.com/zcash/zcash/pull/6646
+
+Note that `zcashd` also requires transactions to pay at least a
+"relay threshold" fee. As part of the ZIP 317 work, this rule was
+simplified for `zcashd` v5.5.0:
+
+* https://github.com/zcash/zcash/pull/6542/files#diff-34d21af3c614ea3cee120df276c9c4ae95053830d7f1d3deaf009a4625409ad2
+
+Deployment in zebra
+-------------------
+
+`zebra` does not provide a wallet, and so does not need to calculate
+ZIP 317 fees in order to construct transactions.
+
+`zebra` v1.0.0-rc.3 implemented the current `Recommended algorithm for
+block template construction`_ in:
+
+* https://github.com/ZcashFoundation/zebra/pull/5724
+* https://github.com/ZcashFoundation/zebra/pull/5776 (algorithm update)
+
+`zebra` v1.0.0-rc.2 had implemented an earlier version of this algorithm.
+The value used for :math:`block\_unpaid\_action\_limit` in `zebra` is not
+configurable.
+
+`zebra` v1.0.0-rc.2 implemented the change to `Mempool size limiting`_ in:
+
+* https://github.com/ZcashFoundation/zebra/pull/5703
+
+`zebra` v1.0.0-rc.8 implemented `Transaction relaying`_ changes in:
+
+* https://github.com/ZcashFoundation/zebra/pull/6556
+* https://github.com/ZcashFoundation/zebra/pull/6585
+
+`zebra` uses a similar relay threshold rule to `zcashd`, but additionally
+enforces a minimum fee of 100 zatoshis (this differs from `zcashd` only for
+transactions of less than 1000 bytes, assuming that `zcashd` uses its
+default value for ``-minrelaytxfee``).
```

Comment:
`zcashd` actually does something weird and discontinuous below 1000 bytes (I'll file an issue).

Reply:
https://github.com/zcash/zcash/issues/6682